core/src/test/scala/tech/beshu/ror/utils/TestsUtils.scala

```diff
@@ -348,9 +348,9 @@ object TestsUtils {
 
   def jsonPathFrom(value: String): JsonPath = JsonPath(value).get
 
-  def uriFrom(value: String): Uri = Uri.parse(value).get
+  def uriFrom(value: String): Uri = Uri.parse(value).toOption.get
```

Comment:
the exception context is missed when you convert toOption and then call `get`

Reply:
fixed

Reply:
fixed